.github/workflows/test.yml

```diff
@@ -84,13 +84,13 @@ jobs:
         install-mlx: [0]
         install-xarray: [0]
         part:
-          - "tests --ignore=tests/tensor --ignore=tests/scan --ignore=tests/xtensor"
+          - "tests --ignore=tests/scan --ignore=tests/tensor --ignore=tests/xtensor"
```

Comment:
This removes the `tensor/test_inplace.py` and reorders the  ignore / parts to be more readible. They ignore show up in the same order where each part is that reintroduced.

May still change after the PR runs to rebalance the workload so they take more or less the same time

Reply:
Jobs look well balanced